spec/support/notifications_helper.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module NotificationsHelper
+  EVENTS = %w[rdv_created rdv_cancelled rdv_updated rdv_upcoming_reminder].freeze
```

Comment:
Je pense qu'on nest face à un bon exemple d'usage de `Symbol` plutôt que de `String`. 

En effet, ces noms sont des références à des sortes d'enums que l'on utilise à travers le code. Ils ne représentent pas des données, mais des concepts.

```suggestion
  EVENTS = %i[rdv_created rdv_cancelled rdv_updated rdv_upcoming_reminder].freeze
```

Et donc idem dans le reste du fichier : 

```ruby
    case event
    when :rdv_created
    #...
```

Et idem pour `"mail"` et `"sms"`, qui peuvent devenir `:mail` et `:sms`.

Reply:
C'est fait :) merci